app/containers/ReactionsList/AllTab.tsx

```diff
@@ -25,15 +26,25 @@ interface IAllTabProps {
 
 const AllReactionsListItem = ({ item, baseUrl, getCustomEmoji, username }: IAllReactionsListItemProps) => {
 	const { colors } = useTheme();
+	const useRealName = useSelector((state: IApplicationState) => state.settings.UI_Use_Real_Name);
```

Comment:
After I react, it doesn't apply use real name anymore

https://user-images.githubusercontent.com/804994/187958874-470ec04e-a409-4b68-97f6-d2370b9e5329.mp4

Reply:
I could reproduce this on web app as well so probably an issue with the API.